backend/services/file_service.py

```
@@ -0,0 +1,31 @@
+
+def save_upload_file(upload_file: UploadFile, destination_dir: str, filename: str) -> str:
+    file_path = os.path.join(destination_dir, filename)
+    with open(file_path, "wb") as buffer:
+        shutil.copyfileobj(upload_file.file, buffer)
+    return file_path
+
+
+def generate_secure_filename(user_id: int, file_ext: str) -> str:
+    return f"{user_id}_{secrets.token_hex(8)}{file_ext}"
+
+def remove_old_file_if_exists(file_path: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `remove_old_file_if_exists`

Add return type annotation: `None`

(ANN201)